src/implementations/vanilla/components/ordered-list/ordered-list.scss

```diff
@@ -40,6 +40,10 @@ $list: null !default;
   &:last-of-type {
     margin-bottom: var(--s-xs);
   }
+
+  .ecl-link {
+    text-decoration: none;
```

Comment:
shouldn't we use standalone links here instead of forcing the display? There may be links inside paragraphs that would still need to be underlined

Reply:
ehm..we do not include the link component here, we expect markup to be passed to the template, instead, and this would mean that the users would have to add the ecl-link--standalone every time they want to have a link.
I do not understand the reason for this change, but i'd say that if the expected look and feel with links is without the underline better doing this than introducing a breaking change.